susemanager-utils/susemanager-sls/modules/pillar/suma_minion.py

```diff
@@ -100,7 +100,10 @@ def ext_pillar(minion_id, *args):
         data_filename = os.path.join(MANAGER_PILLAR_DATA_PATH, minion_pillar_filename_prefix + suffix)
         if os.path.exists(data_filename):
             try:
-                ret.update(yaml.load(open(data_filename).read(), Loader=yaml.FullLoader))
+                ret = salt.utils.dictupdate.merge(
```

Comment:
why is this needed here?

Reply:
There may be multiple beacons configured in different files and we don't want one to kill the other. Typically here I had:

```yaml
- beacons:
    pkgset:
      cookie: /var/cache/salt/minion/rpmdb.cookie
      interval: 5
```

That was kicked out by the virtpoller config